src/Block/Fetch.hs

```diff
@@ -291,13 +304,13 @@ fetchNextBlock = do
         $ Text.pack $ "Error decoding RequestNext response: " <> e
     Right (RollBackward _point _tip) ->
       logWarnNS "fetchNextBlock" "Received RollBackward response"
-    Right (RollForward (UnsupportedBlock type_ raw) _tip) ->
+    Right (RollForward (UnsupportedBlock type_ raw) _tip) -> do
       logWarnNS
         "fetchNextBlock"
         $ "Received unsupported block in the RollForward response with type: "
           <> type_
-          <> " raw: " -- TODO: raw output only on debug logging level
-          <> raw
+      logDebugNS "fetchNextBlock" $
```

Comment:
With the new control of log info, this must be enough to only show `raw` in `debug` mode